pkg/store/copr/batch_coprocessor.go

```diff
@@ -1134,9 +1134,13 @@ func (c *CopClient) sendBatch(ctx context.Context, req *kv.Request, vars *tikv.V
 		}
 		tasks, err = buildBatchCopTasksForPartitionedTable(ctx, bo, c.store.kvStore, keyRanges, req.StoreType, false, 0, false, 0, partitionIDs, tiflashcompute.DispatchPolicyInvalid, option.TiFlashReplicaRead, option.AppendWarning)
 	} else {
-		// TODO: merge the if branch.
-		ranges := NewKeyRanges(req.KeyRanges.FirstPartitionRange())
-		tasks, err = buildBatchCopTasksForNonPartitionedTable(ctx, bo, c.store.kvStore, ranges, req.StoreType, false, 0, false, 0, tiflashcompute.DispatchPolicyInvalid, option.TiFlashReplicaRead, option.AppendWarning)
+		if req.StoreType == kv.TiFlash && req.FullText {
```

Comment:
I think it's conflicting with the possible adjustment in `adjustReadReqType`?

If the request is directly set as `fulltext`, is it possible to use mpp request instead of batchcop?

Reply:
indexlookup now only support cop/batchcop, I think....